test/unit/io/structure_file/structure_file_output_test.cpp

```diff
@@ -5,6 +5,7 @@
 // shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
 // -----------------------------------------------------------------------------------------------------
 
+#include <cstdio>
```

Comment:
for what do you need this include?

Reply:
I had used std::printf before, but deleted it, so the header is not needed anymore. Well spotted!